src/components/RichTextEditor/EditorPositioningWrapper/index.ts

```diff
@@ -10,15 +10,15 @@ import {
 export { Position } from './types';
 export const EditorPositioningWrapper: EditorPositioningWrapperProps = {
```

Comment:
This you can make function which gets two params border and position and returns the classes you need.

Reply:
fixed